recipes-extended/spi-minnowmax-board/files/ostro-6lowpan.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=6loWPAN network setup
+Before=connman.service
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+EnvironmentFile=-/etc/ostro-6lowpan.conf
```

Comment:
This file must be provided, right? Or can we auto create these values otherwise?

If it must be provided, we can remove the "-" to make the file optional and add ConditionPath

But my preference is to keep this and inside the scripts to auto generate a hwaddr and use a fixed set of others. This way we can simplify the setup of a mesh of ostro devices.